main.tex

```diff
@@ -506,19 +506,23 @@ \subsection*{Cross-resistance between pyrethroids and DDT}
 %
 The \texttt{L995F} and \texttt{L995S} alleles are known to increase resistance to both of these insecticide classes \cite{Martinez-Torres1998,Ranson2000}.
 %
-Except in a few locations, DDT has not been used for IRS within the last two decades, and is not suitable for use in bed-nets due to potential carcinogenic effects \cite{Turusov2002}.
+By 2012, over half of African households owned at least one pyrethroid impregnated ITN and nearly two thirds of IRS were programmes using pyrethroids \cite{Hemingway2016}. 
+%
+Pyrethroids were also introduced into agriculture in Africa prior to the scale-up of public health vector control programmes, and continue to be used on a variety of crops such as cotton \cite{Reid2016}.
+%
+DDT was used in Africa for several pilot IRS projects carried out during the first global campaign to eradicate malaria, during the 1950s and 1960s \cite{Davies2007b}.
 %
-DDT was, however, used in Africa for several pilot IRS projects carried out during the first global campaign to eradicate malaria, during the 1950s and 1960s \cite{Davies2007b}.
+Unlike pyrethroids, however, DDT has not been widely used for IRS within the last two decades.
```

Comment:
Still think we need to qualify this a bit, DDT is still used for IRS in quite a few locations from what I've seen from Penny's maps. E.g., how about:

"DDT is still approved for IRS use by WHO and remains in use in some locations, however within the last two decades pyrethroid use has been far more common and widespread."